doc/conftest.py

```diff
@@ -46,23 +46,23 @@ def setup_working_with_text_data():
 def setup_compose():
     try:
         import pandas  # noqa
-    except ImportError:
-        raise SkipTest("Skipping compose.rst, pandas not installed")
+    except ImportError as e:
+        raise SkipTest("Skipping compose.rst, pandas not installed") from e
```

Comment:
Raising a SkipTest is something that pytest will understand, I'm not sure we need the `from` construct here

Reply:
Fixed.